kdestandardize.ado

```diff
@@ -1369,10 +1378,26 @@ prog def kdestandardize
 
 	
 	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
+	// Handles instances of the  variable EQUITY_MEASURE
+	if `: list posof "equity_measure" in x' != 0 {
+	//Recodes equity measure variable with numeric values
+		qui: replace equity_measure=cond(`rx'(equity_measure, "Community Support and Involvement Composite", 1), "1", ///
+									cond(equity_measure=="Managing Student Conduct Composite","2",  ///
+									cond(equity_measure=="Overall Effectiveness of School Teachers and Leaders","3",  ///
+									cond(equity_measure=="Overall Student Growth Rating of Teachers and Leaders","4", ///
+									cond(equity_measure=="Percentage of new and Kentucky Teacher Internship Program (KTIP) teachers","5", ///
+									cond(equity_measure=="Percentage of Teacher Turnover","6", ///
+									cond(equity_measure=="School Leadership Composite","7")))))) ///
```

Comment:
formatting to use when testing for equality with :

variableName == value

Include space after commas when used to delimit arguments/parameters being passed to functions.  
Condition function requires three arguments:

cond(test, valueIfTrue, valueIfFalse)

```
// This fails on execution and needs formatting
cond(equity_measure=="School Leadership Composite","7")

// This would execute in place of the previous line and is easier to read:
cond(equity_measure == "School Leadership Composite", "7", "")

```